packages/webpack-config/src/postcss.ts

```diff
@@ -0,0 +1,75 @@
+import type { AcceptedPlugin } from "postcss";
+import postcssCalc from "postcss-calc";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-require-imports
+const postcssPresetEnv = require("postcss-preset-env");
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-require-imports
+const postcssMixins = require("postcss-mixins");
```

Comment:
❓ Is there a reason why we cannot use `import` syntax for `postcssPresetEnv` and `postcssMixins`?

Reply:
I was able to use `@types/postcss-mixins` to import it, but the types for `postcss-preset-env` are only for version 8.

Reply:
Milan pushed a commit to remove the second `include`.